### PR TITLE
chore(docs): adds remark-lint, fix #1741

### LINKED ISSUE #1741: Setup remark-lint for the docs
## Feature request

As we have discussed with @sarahdayan the future roadmap of `remark-lint` adoption is:
1. I will set up the initial `remark`-based config at https://github.com/vuejs/vuepress/tree/master/packages/docs (estimate: 6.08.2019)
2. I will have to figure out what style rules does this repo already have, so the future diffs would be minimal (+1-2 days)
3. I will fix the style violations and mark PR as ready to be reviewed
4. Review, fixes, happiness 

Later we can add more awesome plugins that enforces some best-practices like:
1. Dead links checker: https://github.com/davidtheclark/remark-lint-no-dead-urls
2. List of all plugins is here: https://github.com/remarkjs/remark-lint#list-of-external-rules

There's also an auto-formatter available: https://github.com/remarkjs/remark-lint#using-remark-to-fix-your-markdown (not sure about it, since I have never used it)

#### What problem does this feature solve?

It simplifies the documentation writing/refactoring/reviewing process by reducing the cognitive complexity on the reader and the writer. 

#### Are you willing to work on this yourself?

Yeap, but any help is appreciated. 🙂

### PR DESCRIPTION
**Pull request by sobolevn**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Code style update
- [x] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature
- [x] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by sobolevn** [comment]
It works with the minimal changes from my side.

```
» npx lerna run lint-md:style
lerna notice cli v3.13.4
lerna info Executing command in 1 package: "yarn run lint-md:style"
lerna info run Ran npm script 'lint-md:style' in 'docs' in 1.7s:
yarn run v1.9.4
$ remark --quiet --frail .
Done in 1.32s.
lerna success run Ran npm script 'lint-md:style' in 1 package in 1.7s:
lerna success - docs
```

File: packages/docs/.remarkrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    "preset-lint-recommended",
+    "preset-lint-consistent",
+
```
Comment (sarahdayan) [inline packages/docs/.remarkrc]: I would also recommend this great plugin: https://github.com/remarkjs/remark-lint

File: packages/docs/package.json [hunk 2]
```diff
@@ -9,7 +9,9 @@
     "build": "vuepress build docs --temp .temp",
     "view-info": "vuepress view-info docs --temp .temp",
     "show-help": "vuepress --help",
-    "lint-md": "textlint ./docs/**/*.md"
+    "lint-md:wording": "textlint ./docs/**/*.md",
+    "lint-md:style": "remark --quiet --frail .",
+    "lint-md": "pwd"
```
Comment (sarahdayan) [inline packages/docs/package.json]: Why do we want to print the working directory? The precommit hook currently runs this command so we're no longer going to run the linting commands... unless I'm missing something 😀

Comment (sobolevn) [reply packages/docs/package.json]: @sarahdayan oops!

File: packages/docs/.remarkrc (hunk 1 above)
Comment (sobolevn) [reply packages/docs/.remarkrc]: It is already included: https://github.com/remarkjs/remark-lint/blob/master/packages/remark-preset-lint-consistent/index.js#L10

Comment (sarahdayan) [reply packages/docs/.remarkrc]: Ah, my bad! 🤦‍♀

**Pull request merged**